week-1/exercises/count.js

```diff
@@ -1,6 +1,11 @@
 let count = 0;
 
-count = count + 1;
+count = count + 1; 
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+
+count = count + 1; //line three is increasing 0 by 1 and the =  is the operator here.S
```

Comment:
The = symbol is the assignment operator in JavaScript, and it is used to assign a value to a variable.

In this specific line, the assignment operator = assigns the result of the addition back to the count variable.